src/js/containers/dashboard/CreatedByContainer.jsx

```diff
@@ -45,7 +45,7 @@ class CreatedByContainer extends React.Component {
   loadData() {
     if (this.props.createdByList.createdBy.length === 0) {
```

Comment:
When switching between FABS and DABS dashboard, `createdByList` is already populated by the first table that was viewed, so `fetchCreatedBy` doesn't get called with the correct param